services/ui-src/src/components/pages/Dashboard/DashboardTable.tsx

```diff
@@ -66,7 +66,11 @@ export const HorizontalTable = (props: TableProps) => {
           {props.showEditNameColumn && (
             <Td fontWeight={"bold"}>
               <button onClick={() => props.openAddEditReportModal(report)}>
-                <Image src={editIcon} alt="Edit Report Name" minW={"1.75rem"} />
+                <Image
+                  src={editIcon}
+                  aria-label={"Edit " + report.name + " report name"}
```

Comment:
I've got an annoying nit here - would you mind swapping out the string concatenation for string interpolation? I try to stick to interpolation because there are cases where it's a lot more readable.

```suggestion
                  aria-label={`Edit ${report.name} report name`}
```

Reply:
good suggestion, thank you!